arrayLib/lib/arrayLib2.js

```diff
@@ -0,0 +1,114 @@
+var arrayLib2 = (function () {
+  'use strict'
+
+  var methods = {}
+
+  function _bubbleSort (arr) {
+    var len = arr.length
+    for (var i = 0; i < len - 2; i++) {
+      for (let j = 0; j < len - i - 1; j++) {
+        if (arr[j] > arr[j + 1]) {
+          _swap(j)
+        }
+      }
+    }
+
+    function _swap (j) {
+      arr[j] = arr[j] + arr[j + 1]
+      arr[j + 1] = arr[j] - arr[j + 1]
+      arr[j] = arr[j] - arr[j + 1]
+    }
+  }
+
+  function _insertionSort (arr) {
+    for (var i = 1; i < arr.length; i++) {
+      var x = arr[i]
+      var j = i
+      while (j >= 1 && arr[j - 1] > x) {
+        arr[j] = arr[j - 1]
+        j = j - 1
+      }
+      arr[j] = x
+    }
+  }
+
+  methods.sort = function (arr, name) {
+    if (arr === undefined || arr.length === 0) return []
+    if (name === undefined) return arr
```

Comment:
if a user haven't passed the name you should throw an error so that they know that the sort wasn't successfull. It's confusing tho, when you pass in an array and it returns the same array without any errors. 
I'd suggest making 'bubble' as default.